astro/src/content/docs/apis/identity-providers/linkedin.mdx

```diff
@@ -32,16 +32,26 @@ export const idp_linking_strategy = 'LinkByEmail';
 This API has been available since 1.23.0
 </Aside>
 
-The LinkedIn identity provider type will use OAuth 2.0 to authenticate a user with LinkedIn. It will also provide a `Login with LinkedIn` button on FusionAuth's login page that will direct a user to the LinkedIn login page. Additionally, after successful user authentication, this identity provider will call LinkedIn's `/v2/me` and `/v2/emailAddress` APIs to load additional details about the user and store them in FusionAuth.
+The LinkedIn identity provider type will use OAuth 2.0 to authenticate a user with LinkedIn. It will also provide a `Login with LinkedIn` button on FusionAuth's login page that will direct a user to the LinkedIn login page. Additionally, after successful user authentication, this identity provider will either call LinkedIn's `/v2/userinfo` API or their `/v2/me` and `/v2/emailAddress` APIs to load additional details about the user and store them in FusionAuth.
 
-The email address returned by the LinkedIn `/v2/emailAddress` API will be used to create or look up the existing user. Additional claims returned by LinkedIn can be used to reconcile the User to FusionAuth by using a LinkedIn Reconcile lambda. Unless you assign a reconcile lambda to this provider, only the `email` address will be used from the available claims returned by LinkedIn.
+The email address returned by the LinkedIn API will be used to create or look up the existing user. Additional claims returned by LinkedIn can be used to reconcile the User to FusionAuth by using a LinkedIn Reconcile lambda. Unless you assign a reconcile lambda to this provider, only the `email` address will be used from the available claims returned by LinkedIn.
 
 <TokenStorageNote idp_long_lived_token_type="access_token"
                   idp_display_name={idp_display_name}
                   idp_linking_strategy={idp_linking_strategy}
                   token_name="access_token"
                   return_text="login endpoint" />
 
+<Aside type="caution">
```

Comment:
This is kind of long for an aside. Maybe we should put this under a `Support for Different LinkedIn APIs` heading or something like that.